app/services/application/ApplicationService.scala

```diff
@@ -71,4 +72,47 @@ trait ApplicationService extends EventSink {
       case None => throw ApplicationNotFound(applicationId)
     }.map(_ => ())
   }
+
+  def confirmAdjustment(applicationId: String, adjustmentInformation: AdjustmentManagement)
+                       (implicit hc: HeaderCarrier, rh: RequestHeader): Future[Unit] = {
+
+    val standardEventList = DataStoreEvents.ManageAdjustmentsUpdated(applicationId) ::
+      AuditEvents.AdjustmentsConfirmed(Map("applicationId" -> applicationId, "adjustments" -> adjustmentInformation.toString)) ::
+      Nil
+
+    def toEmailString(header: String, adjustmentDetail: Option[AdjustmentDetail]): String ={
+      def mkString(ad: Option[AdjustmentDetail]): Option[String] = {
+        ad.map { e =>
+          val et = e.timeNeeded.map( tn => s"$tn% extra time")
+          val iv_info = e.invigilatedInfo
```

Comment:
what is going on with the naming conventions? Somebody been programming Ruby recently?